mailbox/elasticsearch/src/test/resources/logback-test.xml

```diff
@@ -21,5 +21,7 @@
                 <appender-ref ref="CONSOLE" />
         </logger>
 
+        <logger name="com.datastax.driver.core.QueryLogger.SLOW" level="DEBUG" />
```

Comment:
> My question is how do we know the order of the different maven project in the classpath ? Do we really control that properly ?

Nope, no control, and you can't know.

Thanks for raising the concern. We should have it in the dockerfiles folder updated. For tests I think we care a bit less about slow queries (but I see no problem updating that everywhere)